ema/src/Ema/CLI.hs

```diff
@@ -26,9 +26,17 @@ data Action
     -- of generated files.
     Generate FilePath
   | -- | Run the live server
-    Run (Host, Maybe Port)
+    Run (Host, Maybe Port, NoWebSocket)
   deriving stock (Eq, Show, Generic)
 
+-- | Whether to disable websocket-based refresh and page loads.
+newtype NoWebSocket = NoWebSocket {unNoWebSocket :: Bool}
+  deriving newtype (Eq, Show)
+  deriving stock (Generic)
```

Comment:
I'd much prefer to propagate `EmaWebSocketOptions r` to the `CLI.Action` type ... but the `r` parameter gets in the way.